modules/contracts/src.ts/services/ethReader.ts

```diff
@@ -40,6 +40,7 @@ import { ChannelFactory, ChannelMastercopy, TransferDefinition, TransferRegistry
 import { Evt } from "evt";
 
 export const MinGasPrice = parseUnits("5", "gwei");
+export const BumpGasPrice = 30; // 30% bump gas price
```

Comment:
Let's use the normal convention we use for these constants which would be `MIN_GAS_PRICE` and `BUMP_GAS_PRICE`.

Reply:
gotchha